hazelcast/src/main/java/com/hazelcast/map/impl/proxy/MapProxySupport.java

```diff
@@ -1144,7 +1144,7 @@ private static void validateEntryProcessorForSingleKeyProcessing(EntryProcessor
      * Async version of {@link #executeOnKeysInternal}.
      */
     public ICompletableFuture<Map<K, Object>> executeOnKeysInternalAsync(Set<K> keys, Set<Data> dataKeys,
-                EntryProcessor entryProcessor, final ExecutionCallback<Map<K, Object>> callback) {
+                EntryProcessor entryProcessor) {
```

Comment:
Can we now make `exeucteOnKeysInternal` forward to this method and get rid of the `TODO`s in lines 1117-1120?